internal/pipeline/pipeline.go

```diff
@@ -37,6 +38,7 @@ type Piper interface {
 // BuildPipeline contains all build-related pipe implementations in order.
 // nolint:gochecknoglobals
 var BuildPipeline = []Piper{
+	gomod.Pipe{},           // setup gomod-related stuff
```

Comment:
Wouldn't some `GO*` environment variables potentially affect the output of this, i.e. are you sure the order is correct here? 🤔

Reply:
afaik no 🤔 

if something appears we can change though